packages/terra-framework-docs/src/terra-dev-site/doc/embedded-content-consumer/EmbeddedContentConsumer.1.doc.mdx

```diff
@@ -56,6 +57,23 @@ Consumer.init();
 * [Responsive Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#responsive-support)
 * [Mobile Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#mobile-support)
 
+## Accessibility
+
+<Notice variant="important" ariaLevel="3">
+
+#### Accessibility Guidance: Title
+
+For the purposes of accessibility compliance, it is **critical** to screen reader users that the title of the frame is set to a meaningful title for the content inside the iframe.
+
+#### Accessibility Guidance: Scrolling
+
+When using this Embedded Content Consumer component to embed another page, the default behavior is set to auto resize the content such that the content will display fully inside of the iframe. In scenario where custom configuration of XFC is made to set a `fixedWidth` or `fixedHeight` of the content, the content may not be fully visible in the viewport. For keyboard only users, it is critical that users can navigate to the iframe, and scroll to view the content fully.
+
+When specifying a `fixedWidth` or `fixedHeight` [attributes for XFC](https://github.com/cerner/xfc#iframe-resizing-config), it's also necessary to set `scrolling` option to `true` so that the content can be scrolled when not fully displayed. This is required for accessibility for keyboard only users.
```

Comment:
Nit, slight rephrase needed here, something like:

1. "When specifying `fixedWidth` or `fixedHeight` attributes for XFC"
2. "When specifying the `fixedWidth` or `fixedHeight` attributes for XFC"
3. "When specifying a `fixedWidth` or `fixedHeight` attribute for XFC"